docs/sphinx/using/backends/sims/svsims.rst

```diff
@@ -0,0 +1,293 @@
+    - positive integer
+    - Number of CPU threads used for circuit processing. The default value is `8`.
+  * - ``CUDAQ_MAX_CPU_MEMORY_GB``
+    - non-negative integer, or `NONE`
+    - CPU memory size (in GB) allowed for state-vector migration. `NONE` means unlimited (up to physical memory constraints). Default is 0GB (disabled, variable is not set to any value).
+  * - ``CUDAQ_MAX_GPU_MEMORY_GB``
+    - positive integer, or `NONE`
+    - GPU memory (in GB) allowed for on-device state-vector allocation. As the state-vector size exceeds this limit, host memory will be utilized for migration. `NONE` means unlimited (up to physical memory constraints). This is the default.
+
+
+
+Single-GPU (fp64) 
+++++++++++++++++++++++++
+
+.. _nvidia-fp64-backend:
+
+The :code:`nvidia` backend allows specification of the floating point precision of the state vector data and can be configured to either 
+double (`fp64`) or single (`fp32`) precision. This option can be chosen for the optimal performance and accuracy.
+
+
+By default, :code:`nvidia` will leverage :code:`FP32` floating point types for the simulation. To 
+switch to :code:`FP64`, specify the :code:`--target-option fp64` `nvq++` command line option for `C++` and `Python` or 
+use `cudaq.set_target('nvidia', option='fp64')` for Python in-source target modification instead. 
+
+.. tab:: Python
+
+    .. code:: bash 
+
+        python3 program.py [...] --target nvidia --target-option fp64
+
+    The precision of the :code:`nvidia` target can also be modified in the application code by calling
+
+    .. code:: python 
+
+        cudaq.set_target('nvidia', option='fp64')
+
+.. tab:: C++
+
+    .. code:: bash 
+
+        nvq++ --target nvidia --target-option fp64 program.cpp [...] -o program.x
+        ./program.x
+
+
+.. deprecated:: 0.8
+    The :code:`nvidia-fp64` targets, which is equivalent setting the `fp64` option on the :code:`nvidia` target, 
+    is deprecated and will be removed in a future release.
+
+
+
+Multi-node multi-GPU (mgpu)
++++++++++++++++++++++++++++++++++
+
+.. _nvidia-mgpu-backend:
+
+The :code:`nvidia` target also provides a state vector simulator accelerated with 
+the :code:`cuStateVec` library with support for Multi-Node, Multi-GPU distribution of the 
+state vector.
+
+This target is necessary to scale applications that require a state vector that cannot fit on a single GPU memory.
+
+The multi-node multi-GPU simulator expects to run within an MPI context.
+To execute a program on the multi-node multi-GPU NVIDIA target, use the following commands 
+(adjust the value of the :code:`-np` flag as needed to reflect available GPU resources on your system):
+
+.. tab:: Python
+
+    Double precision simulation:
+
+    .. code:: bash 
+
+        mpiexec -np 2 python3 program.py [...] --target nvidia --target-option fp64,mgpu
+
+    Single precision simulation:
+    
+    .. code:: bash 
+
+        mpiexec -np 2 python3 program.py [...] --target nvidia --target-option fp32,mgpu
```

Comment:
Maybe add something like this to the single GPU option

Reply:
Yes, I meant showing the code line both precision options under the single GPU option